apps/src/music/blockly/blocks/simple2.js

```diff
@@ -130,12 +119,19 @@ export const triggeredAtSimple2 = {
     tooltip: musicI18n.blockly_blockTriggeredTooltip(),
     helpUrl: DOCS_BASE_URL + 'trigger',
   },
-  generator: block =>
-    `
-      Sequencer.newSequence(startPosition, true);
-      Sequencer.startFunctionContext('${block.getFieldValue(TRIGGER_FIELD)}');
-      Sequencer.playSequential();
-    `,
+  generator: ctx => {
```

Comment:
What's the rationale for using `ctx` for a block?

Reply:
We've used it since the very first music PR, and I figured it was borrowed from elsewhere, but after looking again, I don't know where. :)  We've also used `block` a bit and it makes sense to change them all to that in a separate tidy-up.